### DISCUSSION TITLE
Introduce Gift Cards / Gift Subscription Options

### DISCUSSION THREAD

**Discussion by iAnonymous3000** [ente/ente#2706] [General]
I've been thinking about ways to share this great service with my friends and family. It occurred to me that gift cards and subscription options would be fantastic additions to the platform. Here's why I think this could be beneficial:
- Sharing the love: I'd love to gift Ente Photos to my privacy-conscious friends, especially those still using less private cloud services.
- Supporting Ente: As a user, I want to see Ente thrive. More revenue streams could help ensure the service's long-term sustainability.
- Perfect gifts: Ente subscriptions would make great birthday or holiday gifts for my tech-savvy relatives.

Here are some features I'd love to see:
- Digital gift cards: Maybe with cool, privacy-themed designs?
- Gifting subscriptions: The ability to gift 3 months, 6 months, or a year would be awesome.

Looking forward to the discussion!

**Comment by optymistic** [comment]
Late to this, but I 100% think this should be implemented.
